crates/hyperqueue/src/bin/hq.rs

```diff
@@ -111,10 +111,37 @@ enum SubCommand {
     AutoAlloc(AutoAllocOpts),
     /// Event log management
     EventLog(EventLogOpts),
-    ///Commands for the dashboard
+    /// Commands for the dashboard
     Dashboard(DashboardOpts),
     /// Generate shell completion script
     GenerateCompletion(GenerateCompletionOpts),
+    /// Commands for displaying task(s)
+    Task(TaskOpts),
+}
+
+// Task CLI options
+
+#[derive(Parser)]
+pub struct TaskOpts {
+    #[clap(subcommand)]
+    subcmd: TaskCommand,
+}
+
+#[derive(Parser)]
+enum TaskCommand {
+    /// Displays all tasks associated with selected job
```

Comment:
`Display tasks associated with selected job(s)`

It does not have to be all tasks, since they can be filtered :)